src/controllers/user.js

```diff
@@ -35,4 +35,31 @@ export default class userController {
       throw util.send(res);
     }
   }
+
+  static async loginUser(req, res) {
+    try {
+      const { error } = loginValidation(req.body);
+      if (error) {
+        util.setError(400, "Validation Error", error.message);
+        return util.send(res);
+      }
+      const { email, username, password } = req.body;
+      const user = await User.emailExist(email);
+      if (username) {
+        throw res.status(405).send({ message: "Username is not required" });
+      }
+      if (!user) {
+        return res.status(404).send({ message: "Email does not exist." });
+      }
+      const validpass = await bcrypt.compare(password, user.password);
+      if (!validpass) {
+        return res.status(404).send({ message: "Password is not correct!." });
```

Comment:
it should be -> `return res.status(400).json({ status: 400, error: "Password is not correct!." });`

key: research more on error codes